File: backend/backend/src/main/java/com/backend/backend/Communication/Response/GetAllUserResponse.java
package com.backend.backend.Communication.Response;


import javax.persistence.Column;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;
import java.util.UUID;

public class GetAllUserResponse {
    private UUID id;

    @NotBlank
    private String username;

    @NotBlank
    @Email
    private String email;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public GetAllUserResponse(UUID id, String username, String email) {
        this.id = id;
        this.username = username;
        this.email = email;
    }
}

File: ClientApp/app/caffadminapp/src/main/java/com/e/caffadminapp/Network/Response/GetAllUserResponse.kt
package com.e.caffadminapp.Network.Response

import android.provider.ContactsContract.CommonDataKinds.Email
import java.util.*


class GetAllUserResponse {

    private lateinit var id: UUID

    private lateinit var username: String

    private lateinit var email: String

    fun getEmail(): String {
        return email
    }

    fun setEmail(email: String) {
        this.email = email
    }

    fun getId(): UUID {
        return id
    }

    fun setId(id: UUID) {
        this.id = id
    }

    fun getUsername(): String {
        return username
    }

    fun setUsername(username: String) {
        this.username = username
    }

    fun GetAllUserResponse(id: UUID, username: String, email: String) {
        this.id = id
        this.username = username
        this.email = email
    }
}